hermes/Cargo.toml

```diff
@@ -58,6 +58,8 @@ structopt              = { version = "0.3.26" }
 strum                  = { version = "0.24.1", features = ["derive"] }
 tokio                  = { version = "1.26.0", features = ["full"] }
 tower-http             = { version = "0.4.0", features = ["cors"] }
+utoipa = { version = "3.4.0", features = ["axum_extras"] }
```

Comment:
Can you add some whitespases here? to keep them match with the rest of the file. (it's not enforces but good to have)